ripe/atlas/tools/commands/probes.py

```diff
@@ -108,7 +109,7 @@ def add_arguments(self):
         )
 
     def run(self):
-
+        """Main function that will run on the command"""
```

Comment:
I'm not convinced that this is a necessary docstring as `run()` is the main function everywhere.